tsc-responsibilities.md

```diff
@@ -0,0 +1,53 @@
+---
+layout: default
+title: TSC responsibilities
+nav_order: 21
+---
+[//]: # (SPDX-License-Identifier: CC-BY-4.0)
+
+# The Hyperledger Technical Steering Committee (TSC) responsibilities
+
+The [Hyperledger Charter](https://www.hyperledger.org/about/charter) states that the TSC is responsible for:
+
+1. coordinating the technical direction of the Hyperledger Foundation;
+
+2. approving project proposals (including, but not limited to, incubation, deprecation and changes to a project’s charter or scope) in accordance with a project lifecycle document to be developed, approved and maintained by the TSC;
+
+3. creating sub-committees or working groups to focus on cross-project technical issues or opportunities;
+
+4. communicating with external and industry organizations concerning related technical matters;
+
+5. appointing representatives to work with other open source or standards communities;
+
+6. establishing community norms, workflows, or policies for releases;
+
+7. discussing, seeking consensus, and where necessary, voting on technical matters relating to the code base that affect multiple projects; and
+
+8. establishing election processes for Maintainers or other leadership roles in the technical community that are not within the scope of any single project.
+
+## TSC Members
+
+TSC members are expected to:
+
+* Regularly participate in the [TSC Meetings](https://wiki.hyperledger.org/display/TSC/TSC+Meeting+Minutes)
+* Review the [TSC Project Updates](https://wiki.hyperledger.org/display/TSC/TSC+Project+Updates) and comment as needed in a timely manner
```

Comment:
IMO, this should be moved into responsibilities along the lines, and I believe we should add something that reflects checking project health status and making sure that the report is accurate and complete.

Reply:
The first list comes from the charter while this one is from the TSC so I don't think moving it is an option, it would confuse things.

I'm happy to extend what the review of the reports entails but I don't think we can require TSC members "[make] sure that the report is accurate and complete". How would one do that? Of course a quick look at LF Insights might reveal some discrepancy with what the report claims but I don't know how one would ensure accuracy and completeness.